libraries/edge-control-libraries/fieldbus/robotmctask/README.md

```diff
@@ -0,0 +1,72 @@
+# Robot Motion Control Task
+
+Robotmctask is a C++ library of motion control task development that provides library and API to support Robot developer to develop robot application with AI inference engine and EtherCAT protocal.
+
+For detailed information, see the [Introduction](./docs/introduction.md).
+
+# Dependency(Ubuntu):
+
+1. install develop tool
+
+```shell
+sudo apt-get install cmake git build-essential libyaml-cpp-dev libeigen3-dev
+```
+
+2. install plcopen-motion library
+
+```shell
+sudo apt install plcopen-motion-dev plcopen-servo-dev   plcopen-ruckig-dev plcopen-databus-dev plcopen-benchmark-dev
```

Comment:
Need to explain to user how to onboard the ECI APT repo prior to running these install commands.  Can probably link to the ECI documentation.